source/docs/network_filter_fuzzing.md

```diff
@@ -0,0 +1,116 @@
+# Generic network-level filter fuzzers overview
+
+[Generic ReadFilter Fuzzer](https://github.com/envoyproxy/envoy/blob/master/test/extensions/filters/network/common/fuzz/network_readfilter_fuzz_test.cc) and [Generic WriteFilter Fuzzer](https://github.com/envoyproxy/envoy/blob/master/test/extensions/filters/network/common/fuzz/network_writefilter_fuzz_test.cc) can be used to do fuzz tests on network-level filters. For a new network-level filter which we want to harden, we may consider adding it to these one or two of these fuzzers depending on the [filter type](https://github.com/envoyproxy/envoy/blob/master/include/envoy/network/filter.h). 
+For ReadFilter, add it to Generic ReadFilter Fuzzer. For WriteFilter, add it to Generic ReadFilter Fuzzer. For Filter, please add it to both.
+Before adding the new filter into the fuzzers, please make sure the filter is designed to accept untrusted inputs, or ready to be hardened to accept untrusted inputs.
```

Comment:
> On slack, Harvey said that we will not be allowing filters to be checked in without fuzzer coverage as soon as we can implement such a check. So you can make a stronger statement here. How about:
> 
> Network filters need to be fuzzed. Filters come in two flavors, each with their own fuzzer. Read filters should extend the [Generic ReadFilter Fuzzer](...). Write Filters should extend the [Generic WriteFilter Fuzzer](...). Some filters are both raed and write filters: They should extend both fuzzers.

Thanks for pointing this out!
Modified. I used "be added into" instead of "extend" here.